src/main/webapp/index2.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Quick question: What is the difference between "index2.html" and "index.html"? Is "Index2.html" just for testing the map stuff?

Reply:
yes it is for testing, i will rename index2.html to map.html to be more clear about what each page is